doc/spec/source/views/transform_view.rst

```diff
@@ -0,0 +1,25 @@
+.. SPDX-FileCopyrightText: Intel Corporation
+..
+.. SPDX-License-Identifier: BSD-3-Clause
+
+.. include:: ../include/distributed-ranges.rst
+
+.. _transform_view:
+
+===============
+ transform_view
+===============
+
+Interface
+=========
+
+.. doxygenclass:: dr::transform_view
```

Comment:
We support `dr::mhp::views::transform` and `dr::shp::views::transform` and want to document that. All the supported views are of the form: `dr::xhp::views::ViewName` We do not want to document the `dr::xhp::views::ViewName_view` because we don't consistently support it and it may change over time.